src/reducers/BoardPage.js

```diff
@@ -456,9 +460,9 @@ const postprocessGetMoreArticles = (myId, result, usersInfo) => {
       NPush:            each.NPush,
       Title:            each.Title,
       CreateTS:         each.CreateTS ? each.CreateTS : utils.emptyTimeStamp(),
-      UpdateTS:         each.UpdateTS ? each.UpdateTS : utils.emptyTimeStamp(),
+      UpdateTS:         each.UpdateTS ? each.UpdateTS : createTS,
       LastSeen:         each.L ? each.L : utils.emptyTimeStamp(),
-      CommentCreateTS:  each.c ? each.c : utils.emptyTimeStamp(),
+      CommentCreateTS:  each.c && !utils.isNullTimeStamp(each.c) ? each.c : createTS,
```

Comment:
should be `each.c && !utils.isNullTimeStamp(each.c) ? each.c : UpdateTS,` instead?

Reply:
It's also possible when there is no comments.

Reply:
Got it, thanks for the confirmation.